packages/components/src/Chip/Chip.tsx

```diff
@@ -41,19 +42,21 @@ export const Chip = ({
     label,
     heading,
   );
+  const Tag = onClick ? "button" : "div";
 
   return (
-    <Tooltip message={tooltipMessage}>
-      <button
+    <Tooltip message={tooltipMessage} setTabIndex={false}>
+      <Tag
         className={classes}
-        onClick={(ev: React.MouseEvent<HTMLButtonElement>) =>
+        onClick={(ev: React.MouseEvent<HTMLButtonElement | HTMLDivElement>) =>
           onClick && onClick(value, ev)
```

Comment:
Nit: could use optional chaining instead

```suggestion
          onClick?.(value, ev)
```